src/Storefront/Framework/Seo/SeoUrlRoute/SeoUrlUpdateListener.php

```diff
@@ -77,37 +68,4 @@ public function updateLandingPageUrls(LandingPageIndexerEvent $event): void
 
         $this->seoUrlUpdater->update(LandingPageSeoUrlRoute::ROUTE_NAME, array_values($event->getIds()));
     }
-
-    /**
-     * @param list<string> $ids
-     *
-     * @return array<string>
-     */
-    private function getCategoryChildren(array $ids): array
```

Comment:
is there an integration test or something that ensures, that SEO URLs for child categories are still correctly generated?

the `SeoUrlUpdateListenerTest` only checks products, if I saw that correctly